### ISSUE TITLE
:bug: Classic layout style side menu widgets not part of the first side menu item cannot be accessed unless the first stacked widgets are traversed first

### ISSUE THREAD

**Issue by SK4G** [jurialmunkey/skin.arctic.fuse#1093]
### Skin section

Widgets

### Current Behavior

When using classic layout style with stacked navigation mode side menu widgets not part of the first side menu item cannot be accessed unless the first stacked widgets are traversed first. The behavior is to that of a linked list. I'm not sure if this the intended behavior. 

When using classic layout style with combined navigation mode all side menu widgets can be accessed, but sometimes the side menu widgets not part of the first side menu item behave the same as described above. Repeatedly trying to access the inaccessible side menu widgets will eventually crash Kodi. Not sure if it is due to changing from different layout modes and there being skin variable remnants from other navigation modes. 

### Expected Behavior

All side menu widgets can be accessed from all of the side menu items, not just from the first side menu item. This behavior is not present in the `Compound` or `Categories` layout styles, which use complex layout styles.

### Steps To Reproduce

1. Use `Classic Layout`
2. Try to access side menu widgets from any except first side menu item.

### Screenshots and Additional Info

```
Classic layout style with combined navigation mode
2024-10-03 16:44:23.891 T:504738   error <general>: Control 101601 in window 10000 has been asked to focus, but it can't
2024-10-03 16:44:26.208 T:504738   error <general>: Control 102601 in window 10000 has been asked to focus, but it can't
2024-10-03 16:44:35.028 T:504738    info <general>: Skipped 5 duplicate messages..

Classic layout style with stacked navigation mode
2024-10-03 16:56:23.943 T:508926 warning <general>: Skin has invalid include: skinvariables-homemenu-staticitems
2024-10-03 16:56:24.017 T:508926    info <general>: Skipped 2 duplicate messages..
2024-10-03 16:56:24.017 T:508926 warning <general>: Skin has invalid include: 
2024-10-03 16:56:24.017 T:508926   error <general>: Misplaced +
2024-10-03 16:56:24.017 T:508926   error <general>: Error parsing boolean expression + skin.hassetting(indicator.ratingscircle)
2024-10-03 16:56:24.017 T:508926   error <general>: Misplaced +
2024-10-03 16:56:24.017 T:508926   error <general>: Error parsing boolean expression + skin.hassetting(indicator.progresspercent)
2024-10-03 16:56:24.017 T:508926 warning <general>: Skin has invalid include: 
2024-10-03 16:56:24.102 T:508926    info <general>: Skipped 104 duplicate messages..
2024-10-03 16:56:24.102 T:508926 warning <general>: Trying to add unsupported control type 7
2024-10-03 16:56:24.501 T:508926    info <general>: Skipped 18 duplicate messages..
2024-10-03 16:56:24.501 T:508926   error <general>: Control 100601 in window 10000 has been asked to focus, but it can't
2024-10-03 16:56:25.466 T:510600   error <general>: DoWork - Direct texture file loading failed for resource://resource.images.studios.coloured/Paramount Animation.png
2024-10-03 16:56:28.847 T:508926   error <general>: Control 103604 in window 10000 has been asked to focus, but it can't
2024-10-03 16:56:37.200 T:508926    info <general>: Skipped 1 duplicate messages..
2024-10-03 16:56:37.200 T:508926   error <general>: Control 419 in window 10000 has been asked to focus, but it can't
2024-10-03 16:56:37.532 T:508926   error <general>: Control 103601 in window 10000 has been asked to focus, but it can't
2024-10-03 16:56:41.520 T:510600   error <general>: DoWork - Direct texture file loading failed for resource://resource.images.studios.coloured/Paramount Animation.png
2024-10-03 16:56:45.168 T:508926    info <general>: Skipped 1 duplicate messages..
2024-10-03 16:56:45.168 T:508926   error <general>: Control 419 in window 10000 has been asked to focus, but it can't

Categories layout
2024-10-04 01:57:47.008 T:685909 warning <general>: Skin has invalid include: skinvariables-homemenu-spotlight
2024-10-04 01:57:47.106 T:685909    info <general>: Skipped 6 duplicate messages..
2024-10-04 01:57:47.106 T:685909 warning <general>: Skin has invalid include: skinvariables-homemenu-widgets-selector
2024-10-04 01:57:47.116 T:685909 warning <general>: Skin has invalid include: 
2024-10-04 01:57:47.116 T:685909   error <general>: Misplaced +
2024-10-04 01:57:47.116 T:685909   error <general>: Error parsing boolean expression + skin.hassetting(indicator.ratingscircle)
2024-10-04 01:57:47.116 T:685909   error <general>: Misplaced +
2024-10-04 01:57:47.116 T:685909   error <general>: Error parsing boolean expression + skin.hassetting(indicator.progresspercent)
2024-10-04 01:57:47.116 T:685909 warning <general>: Skin has invalid include: 
2024-10-04 01:57:47.167 T:685909    info <general>: Skipped 104 duplicate messages..
2024-10-04 01:57:47.167 T:685909   error <general>: Misplaced [
2024-10-04 01:57:47.167 T:685909   error <general>: Error parsing boolean expression [integer.isequal(window.property(tmdbhelper.widgetcontainer),301) + ![$exp[exp_hubs_spotlight_hasitems]]
2024-10-04 01:57:47.171 T:685909 warning <general>: Trying to add unsupported control type 7
2024-10-04 01:57:47.756 T:688822    info <general>: Skipped 46 duplicate messages..
2024-10-04 01:57:47.756 T:688822   error <general>: GetDirectory - Error getting library://music/artists.xml/
2024-10-04 01:57:47.756 T:688822   error <general>: GetDirectory - Error getting library://music/genres.xml/
2024-10-04 01:57:47.756 T:688822   error <general>: GetDirectory - Error getting library://music/years.xml/
2024-10-04 01:57:48.344 T:689464    info <general>: CPythonInvoker(188, /home/luiz/.kodi/addons/plugin.video.themoviedb.helper/resources/script.py): waiting on thread 139827072718528
2024-10-04 01:57:48.499 T:689519    info <general>: CPythonInvoker(191, /home/luiz/.kodi/addons/plugin.video.themoviedb.helper/resources/script.py): waiting on thread 139826401629888
2024-10-04 01:57:53.207 T:688870   error <general>: DoWork - Direct texture file loading failed for resource://resource.images.studios.coloured/Paramount Animation.png
2024-10-04 01:57:55.452 T:685909   error <general>: Control 300 in window 10000 has been asked to focus, but it can't
```

### Checklist

- [X] I have made a separate issue for each bug that I am reporting.
- [X] I have searched the issues and this bug has not been reported.
- [X] I am using the most recent skin version.
- [X] I have confirmed that this bug does not occur in the default skin.
